strfry/plugins/broadcast_vanish_requests.ts

```diff
@@ -19,8 +24,26 @@ function createBroadcastVanishRequests(
     throw new Error("RELAY_URL environment variable is not set.");
   }
 
+  let cache: PubkeyCache;
+  if (!vanishedPubkeyCache) {
+    cache = new PubkeyCache(redis, CACHE_MAX_SIZE);
+    await cache.initialize();
+  } else {
+    cache = vanishedPubkeyCache;
+  }
+
   return async (msg) => {
     const event = msg.event;
+    const pubkey = event.pubkey;
+
+    if (cache.has(pubkey)) {
+      return {
+        id: event.id,
+        action: "shadowReject",
```

Comment:
Why is this "shadow" reject. Can we give a nice error message in this case?

Reply:
I can change it to a normal reject indeed, not real reason to hide that it was deleted